easybuild/easyconfigs/r/REEF3D/REEF3D-25.05_v2-makefile.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/REEF3D-25.05_v2.orig/Makefile b/REEF3D-25.05_v2/Makefile
```

Comment:
@sassy-crick Please add a short description on top of the patch, explaining what it does, and mention source or author as well